docs/src/app/docs/page.tsx

```diff
@@ -3,23 +3,7 @@ import Button from '../../components/button';
 export default function Docs() {
   return (
     <>
-      <h2 className="text-3xl font-bold sm:text-4xl">
-        Welcome to Keystatic{' '}
-        <span className="relative">
-          <svg
-            className="absolute -right-2 bottom-2 w-[115%]"
-            xmlns="http://www.w3.org/2000/svg"
-            fill="none"
-            viewBox="0 0 392 92"
-          >
-            <path
-              fill="#F7DE5B"
-              d="m4.239.201 92.684 2.883 100.722 7.097 99.043 7.211 94.363 2.77-21.813 9.088 14.042 9.919 2.873 8.7-14.795 6.043 7.844 5.477 7.843 5.476-14.691 6.037 11.104 9.535 3.927 10.77-93.59-1.7-100.082-5.647-116.75-3.055-76.39-9.559 12.857-8.312-11.94-9.45 5.534-10.258-4.618-7.502 16.812-1.055L7.21 20.478l5.332-11.703L4.239.201Z"
-            />
-          </svg>
-          <span className="relative">Docs</span>
-        </span>
-      </h2>
+      <h1 className="text-3xl font-bold sm:text-4xl">Keystatic Docs</h1>
```

Comment:
What's the plan for this page? More to come? I can see line 10 is using markdown.

Reply:
There's a ticket for that, I should get round to it eventually 🥴

https://linear.app/keystatic/issue/KEY-119/docs-homepage

I believe the current plan is to redirect this page to a content page called `Installation` which helps users on the first step of their keystatic journey.

Reply:
Awesome!